compiler/one-cmds/tests/prepare_test_materials.sh

```diff
@@ -57,7 +57,7 @@ if [[ ! -s "img_files" ]]; then
     # https://github.com/Samsung/ONE/issues/3213#issuecomment-722757499
 fi
 
-if [ ! -d "raw_files" ] || [ ! -s "datalist.txt" ]; then
+if [ ! -d "raw_files" ] || [ ! -s "datalist.txt" ] || [ ! -d "numpy_files" ] || [ ! -s "datalist_numpy.txt" ]; then
```

Comment:
@jinevening , can we use `datalist.txt` for both raw and numpy files?

Reply:
I think we can't. The names of data files are different for raw and numpy files, so we will need different list files.

Reply:
@Seunghui98 , how is this change related with two python scripts?

Reply:
> how is this change related with merge of two python scripts?

I think you may have felt that the intermediate process for this change is missing.

In fact, the following code should have been included in CMakeLists.txt, in this PR. https://github.com/Samsung/ONE/pull/12283
but by my mistake, I forgot to put the code and uploaded pr, but it was merged. 😅
```
if [ ! -d "numpy_files" ] || [ ! -s "datalist_numpy.txt" ]; then
    ../bin/venv/bin/python preprocess_images_numpy.py
fi
```

Maybe if the code went in without any mistakes, **this change** would have been as follows.
```
if [ ! -d "raw_files" ] || [ ! -s "datalist.txt" ]; then
    ../bin/venv/bin/python preprocess_images.py
fi

if [ ! -d "numpy_files" ] || [ ! -s "datalist_numpy.txt" ]; then
    ../bin/venv/bin/python preprocess_images_numpy.py
fi

>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> merge two scripts

if [ ! -d "raw_files" ] || [ ! -s "datalist.txt" ] || [ ! -d "numpy_files" ] || [ ! -s "datalist_numpy.txt" ]; then
    ../bin/venv/bin/python preprocess_images.py
fi
```

Reply:
I think this change in `CMakelist.txt` is not related to PR content.
so, I will remove this from PR now and upload it in the next PR. :)

Reply:
I think we should land the fix before this PR.

Reply:
@seanshpark 

Okay. i will upload PR.
Thank you a lot :)